### DISCUSSION TITLE
How Would I Make Another URL Unsandboxed?

### DISCUSSION THREAD

**Discussion by PURPL3-CAT** [TurboWarp/extensions#945] [Q&A] [answered]
Is there a way to allow TurboWarp to load extensions from a certain URL unsandboxed? I know I can add them by hand, but I have a list of urls from a http server I would like to load all unsandboxed at once. Do I need to fork and modify TurboWarp/Scratch-Gui?

**Comment by CubesterYT** [comment]
Yes there is, I'll give you the link tomorrow.

**Comment by LilyMakesThings** [comment]
Unfortunately, aside from extensions.turbowarp.org and localhost:8000, there are no other URLs you can load extensions unsandboxed from for safety reasons.

**Comment by CubesterYT** [comment]
You still can, I'll give the method tomorrow.

**Comment by CubesterYT** [comment]
@PURPL3-CAT This is the link: https://turbowarp.org/editor?unsandboxed={any_link_to_your_extension_here}

This link allows you to load any extension from any url in. It only can import one though. But at least it works. (Also remember to remove the {}

**Reply by LilyMakesThings** [reply, marked as answer]
This only works if you have the Extended Gallery (an unofficial userscript created by a few TW ext devs). This does not work in vanilla.

As I said previously, aside from extensions.turbowarp.org and localhost:8000, there are no other URLs you can load extensions unsandboxed from for safety reasons. **The chances of this changing are very slim**.

**Reply by CubesterYT** [reply]
Oof.
